dev-constraints.txt

```diff
@@ -10,5 +10,5 @@ Sphinx==3.1.2
 # development before GA. After GA, we will build against specific releases.
 # Bump the commit frequently during development whenever you are missing
 # changes from upstream.
-opentelemetry-api==0.12b0
-opentelemetry-sdk==0.12b0
+-e git+https://github.com/open-telemetry/opentelemetry-python.git@3cae0775ba12a2f7b4214b8b8c062c5e81002a19#egg=opentelemetry-api&subdirectory=opentelemetry-api
```

Comment:
OOC what is `@3cae0775ba12a2f7b4214b8b8c062c5e81002a19#egg=opentelemetry-api&subdirectory=opentelemetry-api`?

Reply:
It installs opentelemetry-api/sdk packages from the git repo at that commit (`3cae0775ba12a2f7b4214b8b8c062c5e81002a19`) so we can track more closely than the latest release.

Planning to remove it after GA when there are fewer breaking changes: #19